docs/classes/ERC725.md

```diff
@@ -766,86 +975,6 @@ myErc725.decodeMappingKey(
 
 ---
 
-## encodePermissions
```

Comment:
@Hugoo this has not been deleted, just move at the top to sort function names correctly by alphabetical order.